docs/source/user-guide/concepts.rst

```diff
@@ -6,6 +6,48 @@ Concepts
    :local:
    :depth: 1
 
+.. image:: /img/installing-with-conda.png
+    :align: right
+
+.. _installing-with-conda:
+
+Installing with conda
+=====================
+
+Conda install
+-------------
+* When you `conda install` a package that exists in a channel and has no dependencies, conda:
+
+  * looks at your configured channels (in priority)
+
+  * reaches out to the repodata associated with your channels/platform
+
+  * parses repodata to search for the package
+
+  * once the package is found, conda pulls it down and installs
+  
+
+.. _specifying-channels:
+
+Specifying channels
+-------------------
+
+* From the command line use `--channel`
+
+.. code-block:: bash
+
+  $ conda install scipy --channel conda-forge
+
+* From the command line use `--override-channels` to only search the specified channel
```

Comment:
```suggestion
* From the command line use `--override-channels` to only search the specified channel(s), rather than any channels configured in .condarc.  This also ignores conda's default channels.
```